src/charm.py

```diff
@@ -463,6 +464,8 @@ def _on_install(self, event: InstallEvent) -> None:
             return
 
         self._push_default_files()
+        if not self._container.isdir(self._log_dir):
```

Comment:
Do we need to check if the dir exists? Can't we just call the `make_dir` function?

Reply:
I haven't verified that if an image has this dir already, the function doesn't error out. So I checked for it to not be there.